smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -39,30 +39,29 @@
 from ...comm.channel.channel import CommChannelBase
 from ...message_handler import MessageHandler
 from ...mli_schemas.model.model_capnp import Model
-from ..storage.feature_store import FeatureStore, FeatureStoreKey
+from ..storage.feature_store import FeatureStore, ModelKey, TensorKey
 
 if t.TYPE_CHECKING:
-    from smartsim._core.mli.mli_schemas.data.data_references_capnp import TensorKey
     from smartsim._core.mli.mli_schemas.response.response_capnp import Status
     from smartsim._core.mli.mli_schemas.tensor.tensor_capnp import TensorDescriptor
 
 logger = get_logger(__name__)
 
 # Placeholder
-ModelIdentifier = FeatureStoreKey
+ModelIdentifier = ModelKey
 
 
 class InferenceRequest:
```

Comment:
In this file -> InferenceRequest.has_raw_inputs - within the docstring raw_outputs should be raw_inputs

Reply:
fixed